src/components/counter/counter.test.js

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+import { render, screen, fireEvent } from "@testing-library/react";
```

Comment:
Comment relates to these and all other test files - consider using `userEvent` instead of `fireEvent`. Under the hood, `userEvent` calls `fireEvent`, but it also performs actions, that are more user-like. One of the common examples is that clicking the button with `fireEvent` won't focus it, while `userEvent` will.

It's just a standard to perform any interactions with `userEvent`, as it was designed specifically to replicate user-like behaviour, and it's exactly what we want to test